pages/agent/v3/aws/vpc.md.erb

```diff
@@ -36,8 +36,13 @@ access the internet directly. You can use
 [security groups](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html)
 to limit traffic and block inbound network connections to your instances.
 
+<!-- vale Buildkite.h2-h6_sentence_case = NO -->
+<!-- I'm not sure why, but Vale doesn't like the slash here -->
```

Comment:
It's possible that something like 

## Using private subnets for added security 

would be better here?

Reply:
Great. I've been trying to show some restraint and doing the smallest changes that satisfy the linting (otherwise I'd be messing with _everything_) but I'm more than happy to take suggestions beyond that. 707a8a77723fd750f66429acacfdd7db663aa9b5

Reply:
Yeah, we do have to be a little restrained when making these changes. But in this situation I think the / title is enough of an edge case that we can just avoid. :+1: